### ISSUE TITLE
[config] Grid configuration is confusing

### ISSUE THREAD

**Issue by ecryth** [smab/playhouse-web#86]
I wrote down some notes on what it's like trying to reconfigure the grid [here](http://bpaste.net/raw/yKRRig6axVThaauCBtbn/) (Swedish).

Basically, the whole process is a bit confusing. It could be alleviated by:
- Adding a function for removing all lights at the same time (no, changing the grid size is not obvious enough)
- Never changing the active lamp as long as the active lamp has not yet been placed, unless "skip lamp" is used
- Making sure that there is at all times only one lamp turned on, the active lamp

**Comment by FireyFly** [comment]
I agree. Also, as I suggested on IRC I think a legend of the different box colours as well as a short explanation of the setup process would help. I could add that during the next sprint (starting today).

**Comment by wamanuz** [comment]
To clid: I gave you this one during our meeting, but I think it's better to investigate this together on our next meeting.

**Comment by wamanuz** [comment]
It really should save when the configuring is done. I always forget to press the button.
